Gemfile

```diff
@@ -34,13 +34,15 @@ gem 'sprockets', '>= 3.0.0'
 gem 'sprockets-es6'
 
 group :development, :test do
+  gem 'awesome_print', '~> 1.8.0'
   gem 'byebug', platforms: %i[mri mingw x64_mingw]
   gem 'dotenv-rails'
   gem 'erb_lint', require: false
   gem 'faker', '>=1.9.1'
   gem 'i18n-tasks'
   gem 'json_expressions'
   gem 'pry-byebug'
+  gem 'rails-controller-testing'
```

Comment:
Why do you need this gem if your not using `assigns` or `assert_template` in your request spec?  Also we are using request specs to test the behaviour. Both `assert_template` and `assigns` assert the implementation, not the behaviour.